api/src/main/java/com/nicknathanjustin/streamercontracts/donations/OpenDonationDto.java

```diff
@@ -0,0 +1,30 @@
+package com.nicknathanjustin.streamercontracts.donations;
```

Comment:
It might be a good idea to put this in a subfolder of donations...we might have more DTO donation objects in the future (i.e my donations vs other user's donations).